src/ocm/components/clusters/ClustersListToolbar.tsx

```diff
@@ -39,7 +39,7 @@ type ClustersListToolbarProps = {
 };
 
 const clusterStatusFilterLabels = Array.from(
-  new Set(Object.values(omit(CLUSTER_STATUS_LABELS, 'adding-hosts'))),
+  new Set(Object.values(omit(clusterStatusLabels, 'adding-hosts'))),
```

Comment:
I think this also needs to invoke `clusterStatusLabels(t)`. Please check its usage